crates/bevy_pbr/src/light_probe/environment_map.wgsl

```diff
@@ -153,13 +162,36 @@ fn environment_map_light(
     let Edss = 1.0 - (FssEss + FmsEms);
     let kD = diffuse_color * Edss;
 
-
     if (!found_diffuse_indirect) {
         out.diffuse = (FmsEms + kD) * radiances.irradiance;
     } else {
         out.diffuse = vec3(0.0);
     }
 
     out.specular = FssEss * radiances.radiance;
+
+    // Clearcoat
+
+#ifdef STANDARD_MATERIAL_CLEARCOAT
```

Comment:
This file is getting rough 😅 . I won't block on it, but I'd prefer to start factoring it out in multiple functions with this file/other files.

Reply:
It's only 198 lines, which is pretty short. How were you thinking about splitting it up?

Reply:
Splitting the clear coat stuff up to a separate function would help maybe. You're right it's not so bad though (I was looking at it in the diff view).

Reply:
OK, done.